packages/svelte/src/compiler/phases/3-transform/client/visitors/template.js

```diff
@@ -1486,13 +1486,7 @@ function process_children(nodes, parent, { visit, state }) {
 		);
 		const assignment = serialize_template_literal(sequence, visit, state)[1];
 		const init = b.stmt(b.assignment('=', b.member(text_id, b.id('nodeValue')), assignment));
-		const singular = b.stmt(
-			b.call(
-				'$.text_effect',
-				text_id,
-				b.thunk(serialize_template_literal(sequence, visit, state)[1])
-			)
-		);
+		const singular = b.stmt(b.call('$.text_effect', text_id, b.thunk(assignment)));
```

Comment:
the same `serialize_template_literal` call is made two lines above and assigned to `assignment` -- this is just a drive-by optimization I noticed